docs/en/rules/Azure.AppGw.AvailabilityZone.md

```diff
@@ -185,6 +129,7 @@ az network application-gateway create \
 
 ## LINKS
 
-- [Azure template reference](https://docs.microsoft.com/azure/templates/microsoft.network/applicationgateways?tabs=json)
+- [Azure Application Gateway IAC Documentation](https://docs.microsoft.com/azure/templates/microsoft.network/applicationgateways?tabs=json)
```

Comment:
I like to keep this something standard for each rule. I've been moving to **Azure deployment reference** to make it less ARM template specific.

```suggestion
- [Azure deployment reference](https://docs.microsoft.com/azure/templates/microsoft.network/applicationgateways?tabs=json)
```